source/operations/checklists/hardware.rst

```diff
@@ -202,16 +208,85 @@ Storage
+      :class-title: note
+
+      MinIO's strict **read-after-write** and **list-after-write** consistency
+      model requires local drive filesystems.
+
+      MinIO cannot provide consistency guarantees if the underlying storage
+      volumes are NFS or a similar network-attached storage volume. 
+
+      For deployments that *require* using network-attached storage, use
+      NFSv4 for best results.
+
+   Use XFS-Formatted Drives with Labels
+   ++++++++++++++++++++++++++++++++++++
 
    Format drives as XFS and present them to MinIO as a :abbr:`JBOD (Just a Bunch of Disks)` array with no RAID or other pooling configurations.
+   Using any other type of backing storage (SAN/NAS, ext4, RAID, LVM) typically results in a reduction in performance, reliability, predictability, and consistency.
+
+   When formatting XFS drives, apply a unique label per drive.
+   For example, the following command formats four drives as XFS and applies a corresponding drive label.
+
+   .. code-block:: shell
+
+      mkfs.xfs /dev/sdb -L MINIODRIVE1
+      mkfs.xfs /dev/sdc -L MINIODRIVE2
+      mkfs.xfs /dev/sdd -L MINIODRIVE3
+      mkfs.xfs /dev/sde -L MINIODRIVE4
+
+   Mount Drives using ``/etc/fstab``
+   +++++++++++++++++++++++++++++++++
+
+   MinIO **requires** that drives maintain their ordering at the mounted position across restarts.
+   MinIO **does not** support arbitrary migration of a drive with existing MinIO data to a new mount position, whether intentional or as the result of OS-level behavior.
+
+   You **must** use ``/etc/fstab`` or a similar mount control system to mount drives at a consistent path.
+   For example:
+
+   .. code-block:: shell
+      :class: copyable
+
+      $ nano /etc/fstab
+
+      # <file system>        <mount point>    <type>  <options>         <dump>  <pass>
+      LABEL=MINIODRIVE1      /mnt/drive-1     xfs     defaults,noatime  0       2
+      LABEL=MINIODRIVE2      /mnt/drive-2     xfs     defaults,noatime  0       2
+      LABEL=MINIODRIVE3      /mnt/drive-3     xfs     defaults,noatime  0       2
+      LABEL=MINIODRIVE4      /mnt/drive-4     xfs     defaults,noatime  0       2
+
+   You can then use ``mount -a`` or similar automounting configurations to ensure all MinIO drives mount at node startup.
+
+   MinIO **strongly recommends** using label-based mounting rules over UUID-based rules.
+   Label-based rules allow swapping an unhealthy or non-working drive with a replacement that has matching format and label.
+   UUID-based rules require editing the ``/etc/fstab`` file to replace mappings with the new drive UUID.
+
+   .. note:: 
+
+      Cloud environment instances which depend on mounted external storage may encounter boot failure if one or more of the remote file mounts return errors or failure.
+      For example, an AWS ECS instances with mounted persistent EBS volumes may fail to boot with the standard ``/etc/fstab`` configuration if one or more EBS volumes fail to mount.
```

Comment:
If you like: tiny tweak to remove one of the many variations on `fail` here
```suggestion
      For example, an AWS ECS instances with mounted persistent EBS volumes may not boot with the standard ``/etc/fstab`` configuration if one or more EBS volumes fail to mount.
```

Reply:
Note: we'd need to drop `an` or make `instance` singular.